themes/socialblue/components/05-templates/page-node/page-node--sky.scss

```diff
@@ -0,0 +1,11 @@
+@import "settings";
+
+.socialblue--sky {
+  &.path-node {
+    .region--complementary {
+      .block-shariff {
+        display: none;
+      }
+    }
+  }
+}
```

Comment:
Instead of hiding this in CSS should we just not render the block?

Reply:
The problem here is that sharing block should be displayed in a default style (in the sidebar), but in Sky style, it is rendered in a hero template as a field, and we hide a block.
We can add a condition to check if it is Sky style in `social_sharing_block_access`, but before you said that we should avoid adding sky related conditions in modules. 
Since we can't use `hook_block_access` in a theme, we hide it with CSS.

Or do you think a solution to hide it in `theme_preprocess_block()`  will be better? 
See: https://drupal.stackexchange.com/a/230632/17388

Reply:
Okay, lets just leave as is then :)